azure-blob-storage-def.ts

```diff
@@ -0,0 +1,34 @@
+import { BlockBlobClient, StoragePipelineOptions } from "@azure/storage-blob";
+import {
+  AzureNodeConfigWithProxy,
+  AzureNodeStateWithProxy,
+} from "./azure-common-defs";
+import * as nodered from "node-red";
```

Comment:
seeing that you are only using `NodeMessageInFlow`, might as well 
```suggestion
import { NodeMessageInFlow } from "node-red";
```
so that you can use it directly in line 32

Reply:
just a stupid suggestion though